src/redturtle/prenotazioni/restapi/services/booking_info/get.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.restapi.interfaces import ISerializeToJson
+from plone.restapi.services import Service
+from zope.component import getMultiAdapter
+from AccessControl import Unauthorized
+
+
+class BookingInfo(Service):
+    def reply(self):
+        catalog = api.portal.get_tool("portal_catalog")
+
+        query = {"portal_type": "Prenotazione", "UID": self.request.uid}
```

Comment:
Se non c'è uid nella request cosa succede? Si rompe?

Reply:
`curl -i -X GET 'http://localhost:8080/Plone/@prenotazione' -H 'Accept: application/json'

HTTP/1.1 404 Not Found
Content-Length: 102
Content-Type: application/json
Date: Thu, 13 Apr 2023 14:38:18 GMT
Server: waitress
Via: waitress
X-Frame-Options: SAMEORIGIN
X-Powered-By: Zope (www.zope.dev), Python (www.python.org)

{
  "message": "Resource not found: http://localhost:8080/Plone/@prenotazione",
  "type": "NotFound"
}%          `